static/app/views/insights/pages/frontendLandingPage.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  type Filters,
+  useFilters,
+  useUpdateFilters,
+} from 'sentry/views/insights/pages/useFilters';
+import {MODULE_TITLES} from 'sentry/views/insights/settings';
+import {type InsightLandingProps, ModuleName} from 'sentry/views/insights/types';
+
+function WebLandingPage() {
+  const filters = useFilters();
+  const updateFilters = useUpdateFilters();
+
+  const landingPageProps: InsightLandingProps = {disableHeader: true};
+
+  const crumbs: Crumb[] = [
+    {
+      label: t('Performance'),
+      to: '/performance', // There is no page at `/insights/` so there is nothing to link to
+      preservePageFilters: true,
+    },
+    {
+      label: FRONTEND_LANDING_TITLE,
+      to: '/performance/web',
+      preservePageFilters: true,
+    },
+    {
+      label: filters.module ? MODULE_TITLES[filters.module] : 'Overview',
+      to: undefined,
+      preservePageFilters: true,
+    },
+  ];
+
+  const handleTabChange: (key: Filters['module']) => void = key => {
+    if (key === filters.module || (key === 'overview' && !filters.module)) {
+      return;
+    }
+    if (!key) {
+      return;
+    }
+    if (key === 'overview') {
+      updateFilters({module: undefined});
+      return;
+    }
+    updateFilters({module: key});
+  };
+
+  return (
+    <Fragment>
+      <Tabs value={filters.module || 'overview'} onChange={handleTabChange}>
+        <Layout.Header>
+          <Layout.HeaderContent>
+            <Breadcrumbs crumbs={crumbs} />
+
+            <Layout.Title>{FRONTEND_LANDING_TITLE}</Layout.Title>
+          </Layout.HeaderContent>
+          <Layout.HeaderActions>
+            <ButtonBar gap={1}>
+              <FeedbackWidgetButton />
+            </ButtonBar>
+          </Layout.HeaderActions>
+          <TabList>
+            <TabList.Item key="overview">{'Overview'}</TabList.Item>
+            <TabList.Item key={ModuleName.VITAL}>
+              {MODULE_TITLES[ModuleName.VITAL]}
+            </TabList.Item>
+            <TabList.Item key={ModuleName.HTTP}>
+              {MODULE_TITLES[ModuleName.HTTP]}
+            </TabList.Item>
+            <TabList.Item key={ModuleName.RESOURCE}>
+              {MODULE_TITLES[ModuleName.RESOURCE]}
+            </TabList.Item>
+          </TabList>
+        </Layout.Header>
+        <Layout.Body>
+          <Layout.Main fullWidth>
+            <PageAlert />
+            <TabPanels>
+              <TabPanels.Item key="overview">{'overview page'}</TabPanels.Item>
+              <TabPanels.Item key={ModuleName.HTTP}>
+                <HTTPLandingPage {...landingPageProps} />
+              </TabPanels.Item>
+              <TabPanels.Item key={ModuleName.RESOURCE}>
+                <ResourcesLandingPage {...landingPageProps} />
+              </TabPanels.Item>
+              <TabPanels.Item key={ModuleName.VITAL}>
+                <WebVitalsLandingPage {...landingPageProps} />
+              </TabPanels.Item>
+            </TabPanels>
+          </Layout.Main>
+        </Layout.Body>
```

Comment:
Let's delete the `Layout.Body` wrapper component here because it already exists in all the subpages, which is resulting in excessive padding

```suggestion
        <Layout.Main fullWidth>
          <PageAlert />
          <TabPanels>
            <TabPanels.Item key="overview">{'overview page'}</TabPanels.Item>
            <TabPanels.Item key={ModuleName.HTTP}>
              <HTTPLandingPage {...landingPageProps} />
            </TabPanels.Item>
            <TabPanels.Item key={ModuleName.RESOURCE}>
              <ResourcesLandingPage {...landingPageProps} />
            </TabPanels.Item>
            <TabPanels.Item key={ModuleName.VITAL}>
              <WebVitalsLandingPage {...landingPageProps} />
            </TabPanels.Item>
          </TabPanels>
        </Layout.Main>
```

Reply:
Good call!